packages/expo-location/src/Location.ts

```diff
@@ -278,7 +268,7 @@ export async function getForegroundPermissionsAsync(): Promise<LocationPermissio
 // @needsAudit
 /**
  * Asks the user to grant permissions for location while the app is in the foreground.
- * @return A promise that fulfills with an object of type [PermissionResponse](#permissionresponse).
+ * @return A promise that fulfills with an object of type [LocationPermissionResponse](#locationpermissionresponse).
```

Comment:
```suggestion
 * @return A promise that fulfills with an object of type [`LocationPermissionResponse`](#locationpermissionresponse).
```

Same as previous comment.